lib/sportradar/api/nfl/stat.rb

```diff
@@ -0,0 +1,7 @@
+module Sportradar
```

Comment:
Makes sense. If that's the case; perhaps we could just define it as a class and pull all the logic from `StatPack` here, using `Nfl::Stat`  in favor of `Nfl::StatPack`, rather than having an empty module.